### ISSUE TITLE
[FEATURE REQUEST] Colorisation scheme (filenames)

### ISSUE THREAD

**Issue by timoc** [orgzly/orgzly-android#194]
I would like to be able to colorize file-names based on regexp. e.g. 
"CATEGORY_summary text that i like+filetags+topics.org"
with an example regexp:
\([A-Z]+\)*_\([a-z ]*\)
where i can assign a color to regexp match 1 and 2 
It would be nice to have multiple regexps (like emacs mode coloring) to set multiple per CATEGORY colors as in the above example filename. 
This mechanism could also be used for tags, and other colorisations.

**Comment by nevenz** [comment]
You are referring to notebook names displayed in the list of notebooks?

**Comment by timoc** [comment]
Yes in the first instance. I use a filename scheme for notes that is something like the above examples.

But the way i see it, is that if you add this capability (regexp colour) and apply it to the notebook names, then that capability could also be applied to the tag lists, and possibly other metadata that needs highlighting in a document. For example i have in my emacs config a regexp to validate the 'project' tagged headlines use "actionable" project verbs (e.g. create, research etc)
